samples/package.json

```diff
@@ -18,7 +18,10 @@
     "sampletsc": "cd typescript && tsc -p . && cd ..",
     "compile": "npm run tsc && npm run sampletsc",
     "clean": "gts clean && rm -rf build/",
-    "precompile": "npm run clean"
+    "precompile": "npm run clean",
```

Comment:
just confirming you meant to add these commands under samples/package.json and not hte top-level.